css/properties/outline.json

```diff
@@ -43,7 +43,7 @@
               "version_added": "1.2"
             },
             "safari_ios": {
-              "version_added": "3.1"
+              "version_added": "1"
```

Comment:
Is the Safari 1.2 info reliable?

Reply:
Confirmed!  `outline` has an effect in Safari 1.2.